pkg/kernelcache/kext.go

```diff
@@ -2,69 +2,220 @@ package kernelcache
 
 import (
 	"bytes"
+	"encoding/binary"
 	"fmt"
+	"io"
 	"os"
-
-	"github.com/blacktop/go-macho"
-	"github.com/blacktop/go-plist"
 )
 
-type prelinkInfo struct {
-	PrelinkInfoDictionary []CFBundle `plist:"_PrelinkInfoDictionary,omitempty"`
+// KextList lists all the kernel extensions in the kernelcache
+func KextList(kernel string) error {
+	kc, err := NewKernelCache(kernel)
```

Comment:
Check failure on line 13 in pkg/kernelcache/kext.go

 
GitHub Actions
/ build (windows-latest)

pkg/kernelcache/kext.go#L13
undefined: NewKernelCache